modules/fluid_properties/src/utils/BrentsMethod.C

```diff
@@ -34,8 +34,11 @@ bracket(std::function<Real(Real)> const & f, Real & x1, Real & x2)
   if (f1 * f2 > 0.0)
   {
     unsigned int iter = 0;
+    std::stringstream debug_ss;
     while (f1 * f2 > 0.0)
     {
+      debug_ss << "  iteration " << iter << ": (x1,x2) = (" << x1 << "," << x2 << "), (f1,f2) = ("
```

Comment:
that's got to be slow.

maybe only do it in devel
```suggestion
#ifdef DEBUG
      debug_ss << "  iteration " << iter << ": (x1,x2) = (" << x1 << "," << x2 << "), (f1,f2) = ("
```